### ISSUE TITLE
Preformat: Not possible to add multiple lines

### ISSUE THREAD

**Issue by michael** [substance/texture#1219]
It should be possible to edit multi-line preformat blocks. Currently ENTER breaks the node and creates a paragraph underneath. Either we interpret ENTER differently when editing a preformat or at least allow Shift+ENTER to create a new line within the current preformat block.

**Comment by Integral** [comment]
This is not supported by Substance library currently, e.g. [Substance will insert default node type](https://github.com/substance/substance/blob/develop/model/Editing.js#L839-L842).
Newline shift+enter also not supported.

**Comment by Integral** [comment]
I think it would not be a hard thing to change this strategy and make it more configurable. Let's wait for @oliver----.
@michael is that a special thing that we do not use new lines? I think for the `preformat` type or code it is very important feature. Let's look at trivial example:
```
for (var i = 0; i < 9; i++) {
  str = str + i;
}
```
For now you will only be able to input this:
```
for (var i = 0; i < 9; i++) {

  str = str + i;

}
```

**Comment by oliver----** [comment]
This has been solved via #1231
